src/Contracts/DataModels.ts

```diff
@@ -92,6 +92,38 @@ export interface ResourceRequest {
   id: string;
 }
 
+export interface IType {

```

Comment:
If possible, move these definitions in the file where it is being used. We are trying to break this monolithic pattern of putting everything in `DataModels` and `ViewModels`.

Reply:
I initially tried putting everything in JunoClient but then found I needed them in other places. I put them into DataModels as that seem to be the established pattern :p